source/braille.py

```diff
@@ -1398,18 +1406,23 @@ def getFocusContextRegions(obj, oldFocusRegions=None):
 			# No common regions were found.
 			commonRegionsEnd = 0
 			newAncestorsStart = 1
-		# Yield the common regions.
+		# Yield the common regions, thereby setting focusToHardLeft back to False.

```

Comment:
Ah, this was part of my initial implementation of this and I wanted to make sure that all cached regions with focusToHardLeft set to True had it set back to False. But, this isn't necessary.